ui/src/App.vue

```diff
@@ -24,9 +24,6 @@
 </script>
 
 <style>
-  .b-radio.radio {
-    margin-right: 0 !important;
-  }
```

Comment:
This didn't seem to be doing anything other than affect the radios added here so I removed it. I did confirm that the other radios still look the way they should.